fdbserver/Coordination.actor.cpp

```diff
@@ -252,15 +264,27 @@ ACTOR Future<Void> leaderRegister(LeaderElectionRegInterface interf, Key key) {
 	state int clientCount = 0;
 	state Reference<AsyncVar<bool>> hasConnectedClients = Reference<AsyncVar<bool>>( new AsyncVar<bool>(false) );
```

Comment:
Increment and decrement for ElectionResult requests so that we don't shut down while things want a reply.